input/resources/au-medicationdispense.xml

```diff
@@ -78,58 +78,55 @@
       <type>
         <code value="CodeableConcept"/>
       </type>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding">
-      <path value="MedicationDispense.medication[x].coding"/>
-      <slicing>
-        <discriminator>
-          <type value="value"/>
-          <path value="system"/>
-        </discriminator>
-        <rules value="open"/>
-      </slicing>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding:pbs">
-      <path value="MedicationDispense.medication[x].coding"/>
-      <sliceName value="pbs"/>
-      <short value="PBS Item Code"/>
-      <comment
-        value="PBS code from http://pbs.gov.au/code/item. Use of PBS as a code to refer to a type of medication, this excludes implication of context based on the PBS code. Where context is to be implied or stated PBS code needs to be associated with recording a prescription (MedicationRequest) or dispense record (MedicationDispense. )"/>
-      <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org.au/ValueSet/pbs-item"/>
-      </binding>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding:pbs.system">
-      <path value="MedicationDispense.medication[x].coding.system"/>
-      <min value="1"/>
-      <fixedUri value="http://pbs.gov.au/code/item"/>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding:gtin">
-      <path value="MedicationDispense.medication[x].coding"/>
-      <sliceName value="gtin"/>
-      <short value="Medication Package Global Trade Item Number (GTIN)"/>
-      <comment value="GTIN value from http://www.gs1.org/gtin."/>
-      <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org/ValueSet/v3-GTIN"/>
-      </binding>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding:gtin.system">
-      <path value="MedicationDispense.medication[x].coding.system"/>
-      <min value="1"/>
-      <fixedUri value="http://www.gs1.org/gtin"/>
-    </element>
-    <element id="MedicationDispense.medication[x]:medicationCodeableConcept.coding:amt">
-      <path value="MedicationDispense.medication[x].coding"/>
-      <sliceName value="amt"/>
-      <short value="AMT Medicines"/>
       <binding>
-        <strength value="preferred"/>
-        <valueSet value="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1"/>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1"/>
```

Comment:
This link opens to https://tx.ontoserver.csiro.au/fhir/ValueSet/australian-medication-1

Reply:
@JayMurdoch - that's nice. What's the review comment, i.e. what has to change and why?

Reply:
I now understand this is a known issue, and no change is required. Thanks.

Reply:
Please resolve the conversation when you are satisfied.

Reply:
This comment is now resolved.